odxtools/cli/browse.py

```diff
@@ -17,7 +17,7 @@
 # logging.basicConfig(level=logging.DEBUG)
 
 # name of the tool
-name = "browse"
+__name__ = "browse"
```

Comment:
__name__ is an auto generated variable by python itself, it includes the module name

Reply:
sure, but it can be overwritten:
```python
>>> import odxtools.database
>>> odxtools.database.__name__
'odxtools.database'
>>> odxtools.database.__name__ = "bla"
>>> odxtools.database.__name__
'bla'
```

maybe we should rename this to something like `_odxtools_tool_name`...